xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-core/src/main/resources/conf/managed-schema

```diff
@@ -698,6 +698,8 @@ https://solr.apache.org/guide/solr/latest/indexing-guide/fields.html
     </fieldType>
 
     <!-- some examples for different languages (generally ordered by ISO code) -->
+    <!-- Needed to support Locale.ROOT -->
```

Comment:
Would be better to follow the same XWiki: reason "syntax" used in other places to make it easier to match custom XWiki stuff.

Reply:
Ok I missed that, will fix it.